index.js

```diff
@@ -474,5 +485,6 @@ export default createInstance();
 
 export {
 	HTTPError,
-	TimeoutError
+	TimeoutError,
+	NO_RETRY_SYMBOL
```

Comment:
I think it should be named `NO_RETRY` or `noRetry` or `cancel` or `abandon`, etc. Probably shouldn't include "symbol" in its name.

Reply:
I agree, `ABANDON_RETRY` sounds accurate. What do you think?

Reply:
I'm wondering if a more generic name would be preferable. Maybe someone will want to use this symbol in a `beforeRequest` hook to ignore a request. We can always rename it later, though...

Reply:
I don't think it should be uppercase. I like `stopRetry`, or just `stop` if we can to make it generic.

Reply:
Agreed about camel case. I like `abandon` or `abort` the best, but I'm fine with `stop`, too.